src/dynamodb.rs

```diff
@@ -1,6 +1,8 @@
 //! DynamoDB bindings for Rust
 //!
-//! Check [DynamoDBHelper](http://dualspark.github.io/rusoto/rusoto/dynamodb/struct.DynamoDBHelper.html) for convenience functions.
+//! Check
+//! [DynamoDBHelper](http://dualspark.github.io/rusoto/rusoto/dynamodb/struct.DynamoDBHelper.html)
+//! for convenience functions.
```

Comment:
I'm not real enthused about how rustfmt wants these lines.  Not sure if it'll end up mangling the docs on github pages, either.